zink/abi-macro/Cargo.toml

```diff
@@ -20,6 +20,7 @@ serde_json.workspace = true
 zabi = { workspace = true, features = ["bytes", "hex", "selector", "serde", "encoding"] }
 sol-abi.workspace = true
 tiny-keccak = { workspace = true, optional = true, features = [ "sha3", "keccak" ] }
+zint.workspace = true
```

Comment:
zint actually have verbose dependencies, since it's mainly for doc tests, I'd suggest we move this to dev-dependencies